### PR TITLE
add euclid clustering in tote for stow task

### PR DESCRIPTION
**Pull request by knorth55**
related to #1616 

tote_cluster

### CODE REVIEW

**Comment by pazeshun** [comment]
I don't know much about this, so I can't review now.

File: jsk_2016_01_baxter_apc/launch/include/segmentation_each_object_in_tote.launch [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      initial_rot: [0, 0, 0]
+      dimension_x: 0.25
+      dimension_y: 0.45
+      dimension_z: 0.15
+      frame_id: base
+    </rosparam>
+  </group>
+  <group ns="$(arg HAND)_in_tote_extract_pi">
+    <rosparam>
+      max_queue_size: 500
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/launch/include/segmentation_each_object_in_tote.launch]: really need this large queue size?

Comment (knorth55) [inline jsk_2016_01_baxter_apc/launch/include/segmentation_each_object_in_tote.launch]: i just copied from `jsk_2015_05_baxter_apc/recognition_in_hand.launch`, i'm not sure

Comment (wkentaro) [inline jsk_2016_01_baxter_apc/launch/include/segmentation_each_object_in_tote.launch]: You should check it is really necessary.
In recognition_in_hand.launch the large queue_size is used for synchronization of depth and color.
https://github.com/start-jsk/jsk_apc/blob/master/jsk_2015_05_baxter_apc/launch/recognition_in_hand.launch

**Comment by knorth55** [comment]
max_queue_size 500 -> 100, fixed and it works.

File: jsk_2016_01_baxter_apc/launch/include/segmentation_each_object_in_tote.launch [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  <group ns="$(arg HAND)_in_tote_euclid_clustering">
+    <rosparam>
+      tolerance: 0.02
+      min_size: 500
+      max_size: 100000
+    </rosparam>
+  </group>
+  <group ns="$(arg HAND)_in_tote_cpi_decomposer">
+    <rosparam>
+      queue_size: 1000
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/launch/include/segmentation_each_object_in_tote.launch]: @knorth55 What about this queue_size?

Comment (knorth55) [inline jsk_2016_01_baxter_apc/launch/include/segmentation_each_object_in_tote.launch]: 100 would be fine

**Comment by knorth55** [comment]
move to #1636

**Pull request closed without merging**